src/cpp/session/modules/SessionFindTests.cpp

```diff
@@ -99,19 +99,52 @@ TEST_CASE("SessionFind")
       std::string line(kRegexLine);
       size_t replaceMatchOff;
 
-      Replacer replacer(false);
+      Replacer replacer("", false);
```

Comment:
Should the `encoding` be the second parameter with a default value of `""`?

Reply:
That works, I made the update.